docker-compose.yml

```diff
@@ -0,0 +1,271 @@
+      OPENMRS_PORT: ${OPENMRS_PORT:?}
+      OPENMRS_ATOMFEED_USER: ${OPENMRS_ATOMFEED_USER:?}
+      OPENMRS_ATOMFEED_PASSWORD: ${OPENMRS_ATOMFEED_PASSWORD:?}
+      OPENELIS_HOST: ${OPENELIS_HOST:?}
+      OPENELIS_PORT: ${OPENELIS_PORT:?}
+      OPENELIS_ATOMFEED_USER: ${OPENELIS_ATOMFEED_USER:?}
+      OPENELIS_ATOMFEED_PASSWORD: ${OPENELIS_ATOMFEED_PASSWORD:?}
+      ODOO_HOST: ${ODOO_HOST:?}
+      ODOO_PORT: ${ODOO_PORT:?}
+      ODOO_ATOMFEED_USER: ${ODOO_ATOMFEED_USER:?}
+      ODOO_ATOMFEED_PASSWORD: ${ODOO_ATOMFEED_PASSWORD:?}
+    depends_on:
+      odoodb:
+        condition: service_healthy
+
+  openmrs:
+    profiles: ["default","openmrs"]
+    image: bahmniindiadistro/bahmni-openmrs-abdm:${OPENMRS_IMAGE_TAG:?}
+    environment:
+      DB_DATABASE: ${OPENMRS_DB_NAME:?}
+      DB_HOST: ${OPENMRS_DB_HOST:?}
+      DB_USERNAME: ${OPENMRS_DB_USERNAME:?}
+      DB_PASSWORD: ${OPENMRS_DB_PASSWORD:?}
+      DB_CREATE_TABLES: ${OPENMRS_DB_CREATE_TABLES}
+      DB_AUTO_UPDATE: ${OPENMRS_DB_AUTO_UPDATE}
+      MODULE_WEB_ADMIN: ${OPENMRS_MODULE_WEB_ADMIN}
+      DEBUG: ${OPENMRS_DEBUG}
+      OPENELIS_HOST: ${OPENELIS_HOST:?}
+      OPENELIS_PORT: ${OPENELIS_PORT:?}
+      OPENELIS_ATOMFEED_USER: ${OPENELIS_ATOMFEED_USER:?}
+      OPENELIS_ATOMFEED_PASSWORD: ${OPENELIS_ATOMFEED_PASSWORD:?}
+    ports:
+      - "8080:8080"
+    volumes:
+      #- "openmrs-data:/usr/local/tomcat/.OpenMRS/modules"
+      #- "${BAHMNI_OPENMRS_MODULES_PATH:?}/:/usr/local/tomcat/.OpenMRS/modules/"
+      #- "${BAHMNI_CONFIG_PATH:?}/:/etc/bahmni_config/"
+      - "bahmni-patient-images:/home/bahmni/patient_images"
+      - "bahmni-document-images:/home/bahmni/document_images"
+    depends_on:
+      - openmrsdb
+
+  openmrsdb:
+    image: bahmni/openmrs-db:${OPENMRS_DB_IMAGE_TAG:?}
+    restart: always
+    profiles: ["default","openmrs"]
+    environment:
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:?}
+      MYSQL_DATABASE: ${OPENMRS_DB_NAME:?}
+      MYSQL_USER: ${OPENMRS_DB_USERNAME:?}
+      MYSQL_PASSWORD: ${OPENMRS_DB_PASSWORD:?}
+    ports:
+      - "3306:3306"
+    volumes:
+      - "openmrsdbdata:/var/lib/mysql"
+
+  bahmni-web:
+    image: bahmniindiadistro/bahmni-web-abdm:${BAHMNI_WEB_IMAGE_TAG:?}
+    profiles: ["default","openmrs"]
+    #volumes:
+      #- "${BAHMNI_APPS_PATH:?}/ui/app/:/usr/local/apache2/htdocs/bahmni"
+      #- "${BAHMNI_APPS_PATH:?}/ui/node_modules/@bower_components/:/usr/local/apache2/htdocs/bahmni/components"
+      #- "${BAHMNI_CONFIG_PATH:?}/:/usr/local/apache2/htdocs/bahmni_config/"
+
+  db:
+    container_name: db
+    image: "postgres:latest"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=welcome
+    ports:
+      - "5433:5432"
+    volumes:
+      - postgresql:/var/lib/postgresql
+      - postgresql_data:/var/lib/postgresql/data
+  rabbitmq:
+      image: "rabbitmq:3"
+      container_name: rabbitmq
+      ports:
+        - "5672:5672"
+  otp:
+      image: "ganesan92/otp-service"
```

Comment:
Whose account is otp-service being picked from?

Reply:
It came as part of projecteka fork